 -- Errors -- 
2019-02-25 11:07:16 Error Create Table:CREATE TABLE LYY.CLOB1(ID TEXT,TXT TEXT);
错误: 关系 "clob1" 已经存在
2019-02-25 11:07:16 Error Migrate Defination for Table:LYY.CLOB1
org.postgresql.util.PSQLException: 错误: 关系 "clob1" 已经存在
2019-02-25 11:07:17 Error Create Table:CREATE TABLE LYY.CLOB2(ID TEXT,TXT TEXT,IMAGE TEXT);
错误: 关系 "clob2" 已经存在
2019-02-25 11:07:17 Error Migrate Defination for Table:LYY.CLOB2
org.postgresql.util.PSQLException: 错误: 关系 "clob2" 已经存在
2019-02-25 11:07:17 Couldn't Batch Insert Data, then insert row by row.
SQLException: 22023 未设定参数值 3 的内容。
2019-02-25 11:07:17Error Insert One Row: "1",""""dddd"ddlll"""lll"""s",

SQLException: 22023 未设定参数值 3 的内容。
2019-02-25 11:07:17 Error Create Table:CREATE TABLE LYY.CLOB3(ID TEXT,TXT TEXT);
错误: 关系 "clob3" 已经存在
2019-02-25 11:07:17 Error Migrate Defination for Table:LYY.CLOB3
org.postgresql.util.PSQLException: 错误: 关系 "clob3" 已经存在
